cmake/ClangFormat.cmake

```diff
@@ -74,7 +74,7 @@ function(clang_format_add_targets)
     )
     add_custom_target(check-format
       COMMAND ${CMAKE_COMMAND} -E echo "Checking ${formatting_files_length} files..."
-      COMMAND ${CMAKE_SOURCE_DIR}/tools/check_format.sh ${CLANG_FORMAT_EXECUTABLE} ${formatting_files}
+      COMMAND ${CLANG_FORMAT_EXECUTABLE} -style=file --dry-run --Werror ${formatting_files} >/dev/null
```

Comment:
Hmm, wouldn't we want to see the output of the format executable (to identify/fix formatting issues) as opposed to redirecting it to /dev/null?

Reply:
(1) any issues would be output to stderr, not stdout, so outputint stdout to /dev/null won't hide any important information from console.

(2) Having error information at this stage isn't useful, since the user isn't expected to fix these issues manually. This line tripping only means that the user neglected to run `make format` on their code.